phpstan.neon.dist

```diff
@@ -57,3 +59,6 @@ parameters:
 		-
 			identifier: cast.string
 			path: */tests/*
+		-
+			identifier: staticMethod.dynamicCall
```

Comment:
I'm not sure why this issue is being reported for assertion method calls in tests.